AWS/aws/s3/queued.py

```diff
@@ -79,14 +79,18 @@ def get_next_messages(self, **kwargs) -> list[dict]:
     def get_next_objects(self, messages: list) -> Generator[AWSS3Object, None, None]:
         receipts = []
         for message in messages:
-            for record in orjson.loads(message["Body"].encode("utf-8")).get("Records", []):
-                try:
-                    yield AWSS3Object.from_record(self.s3_client, record)
-
-                    if receipt := message.get("ReceiptHandle"):
-                        receipts.append(receipt)
-                except ValueError as e:
-                    self.log_exception(e, message="Invalid record")
+            try:
+                decoded_messages = orjson.loads(message["Body"].encode("utf-8")).get("Records", [])
+                for record in decoded_messages:
+                    try:
+                        yield AWSS3Object.from_record(self.s3_client, record)
+
+                        if receipt := message.get("ReceiptHandle"):
+                            receipts.append(receipt)
+                    except ValueError as e:
+                        self.log_exception(e, message="Invalid record")
+            except ValueError as e:
+                self.log_exception(e, message=f"Invalid JSON in message.\nInvalid message is: {message}")
```

Comment:
The except should probably be at line 84 since it is only covering the json deserialization (same for other instances)